src/esp/assets/ResourceManager.h

```diff
@@ -1110,6 +1077,18 @@ class ResourceManager {
   gfx::ShaderManager shaderManager_;
 
   // ======== Physical parameter data ========
+  /**
+   * @brief Plugin Manager used to instantiate importers which in turn are used
+   * to load asset data
+   */
+  Magnum::PluginManager::Manager<Importer> importManager;
```

Comment:
```suggestion
  Corrade::PluginManager::Manager<Importer> importerManager_;
```

Keeping the PluginManager in namespace where it originated from to avoid confusion, and `importerManager` is commonly used in a lot of Magnum docs and examples so keeping the name the same. 

I had to pause on the `Importer` for a bit, took me a while to realize it's a typedef for `Trade::AbstractImporter`. Again to reduce confusion, maybe remove it and use the original name instead? It's not too long I think ;)